browseable/README.md

```diff
@@ -0,0 +1,101 @@
+# STAC API - Browseable Specification
+
+- [STAC API - Browseable Specification](#stac-api---browseable-specification)
+  - [Link Relations](#link-relations)
+  - [Endpoints](#endpoints)
+  - [Example Landing Page for STAC API - Browseable](#example-landing-page-for-stac-api---browseable)
+  - [Extensions](#extensions)
+
+- **OpenAPI specification:** none
+- **Conformance URIs:** 
+  - <https://api.stacspec.org/v1.0.0-beta.5/browseable>
+  - <https://api.stacspec.org/v1.0.0-beta.5/core>
+- **Extension [Maturity Classification](../extensions.md#extension-maturity):** Pilot
+- **Dependencies**: [STAC API - Core](../core)
+
+A STAC API conforming to the `STAC API - Browseable` conformance class must be structured such that all 
+all Items in the catalog can be accessed by following `child` and `item` link relations. This is a more significant
+constraint than a STAC API without this conformance class or a STAC Catalog that is available over HTTP but does not
+implement STAC API, neither of which have any guarantee regarding the reachability of Items.
+
+Recommendations for structuring Catalogs hierarchically can be found in
+[Structuring Catalog Hierarchies](../core/README.md#structuring-catalog-hierarchies) from the `STAC API - Core` specification.
+
+## Link Relations
+
+This conformance class also requires implementation of the link relations in the [STAC API - Core](../core) conformance class.
+
+Additionally, `child` relations must exist to child Catalogs and Collections and `item` relations to Items, such that
+every Item in the Catalog can be accessed by traversing these relations.
+
+| **rel** | **href** | **From**  | **Description**                        |
+| ------- | -------- | --------- | -------------------------------------- |
+| `child` | various  | STAC Core | The child STAC Catalogs & Collections. |
+| `item`  | various  | STAC Core | The child STAC Items.                  |
+
+Note that there is a different link relation `items` (plural)
+used by the `STAC API - Features` conformance class that links from a collection to the items in
+that collection.
+
+## Endpoints
+
+This conformance class also requires for the endpoints in the [STAC API - Core](../core) conformance class to be implemented.
+
+This conformance class adds no additional endpoints.
+
+## Example Landing Page for STAC API - Browseable
+
+This JSON is what would be expected from an API that implements `STAC API - Browseable`. 
+
+This particular catalog provides both the ability to browse down to child Catalog objects through its
+`child` links, which then will eventually reach Items through `item` link relations.
```

Comment:
The example below doesn't seem particularly meaningful. Could be good to try to illustrate the point more, since usually the example helps show that. Perhaps a little diagram, that shows a non-browsable catalog vs a browsable one? Or perhaps just explain after the example that a catalog not implementing browsable would not have the 'child' links, but it would have them in the 'data' rel link. I think it'd be good to just provide more context, and to help make it clear to existing implementations what they need to do.